components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/util/ActionManagementUtil.java

```diff
@@ -19,15 +19,22 @@
 package org.wso2.carbon.identity.action.management.util;
 
 import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang.StringUtils;
 import org.wso2.carbon.identity.action.management.constant.ActionMgtConstants;
 import org.wso2.carbon.identity.action.management.exception.ActionMgtClientException;
 import org.wso2.carbon.identity.action.management.exception.ActionMgtServerException;
 
+import java.util.regex.Pattern;
+
 /**
  * Utility class for Action Management.
  */
 public class ActionManagementUtil {
```

Comment:
IMO, we should avoid using static util classes as much as possible because you can't unit test. You can write an ActionValidator class. Instantiate at the action service class and use that to validate the actions.

Reply:
+1

Reply:
Updated with b22349fb9192bbbd7f2cf26535baa690d138d6f0